### DISCUSSION TITLE
SMS Error Verizon

### DISCUSSION THREAD

**Discussion by lofosho** [jef/streetmerchant#1185] [Help]
Hello, I'm trying to get my SMS notifications running but keep getting this error  couldn't send sms to ********** for carrier verizon self signed certificate in certificate chain 
Also getting this for e-mail  couldn't send email self signed certificate in certificate chain {

**Comment by Kpa62** [comment]
this is a carrier issue, try sending a email to your cellphone number @vzpix.com. this should trigger a proper text.

**Comment by jef** [comment]
> this is a carrier issue, try sending a email to your cellphone number @vzpix.com. this should trigger a proper text.

I don't think this is the issue here. It probably has to do with the email setup.
